GEM_Workshop/LC_CMS_Workshop/notebook/prepare_train.py

```diff
@@ -0,0 +1,144 @@
+import logging
+from functools import partial
+from typing import Any, List, Tuple
+
+import fs
+import geopandas
+import numpy as np
+from eogrow.core.pipeline import Pipeline
+from eogrow.core.storage import FS
+from eolearn.core import EOTask, EOPatch
+from eolearn.core import LoadTask, SaveTask, EOWorkflow, OverwritePermission, FeatureType, linearly_connect_tasks, parallelize
+from eolearn.core.utils.fs import pickle_fs
+from eolearn.core.utils.fs import unpickle_fs
+from eolearn.geometry.transformations import VectorToRasterTask
+from pydantic import Field
+
+LOGGER = logging.getLogger()
+
+
+class LoadTrainingPolygonsForEOPatch(EOTask):
+    """
+    This task allows the load the training polygons for the EOPatch based on the training for the AOI
+    """
+
+    def __init__(self, raster_feature: Tuple[FeatureType, str], train_polygons_filepath: str, pickeld_fs: bytes):
+        self.raster_feature = raster_feature
+        self.train_polygons_filepath = train_polygons_filepath
+        self.pickled_fs = pickeld_fs
+
+    def execute(self, eopatch: EOPatch):
+        """Execute EOPatch
+            - Transform input geometry to match the DB
+            - Query DB for the vector data
+            - Filter to create train & val geometries based on the feat ids given
+        """
+        # transform to WGS84 CRS
+        # fetch from database & rename `geom` column to `geometry` so that it can be used as EOLearn Vector feature
```

Comment:
are these two comments still relevant?
the one on L39 also seems unnecessary. Could also adjust the docstring a bit since it does no filtering etc. (as is currently described).

Reply:
done